src/components/TransformtoEcommerce.js

```diff
@@ -0,0 +1,128 @@
+                        <p style={{ fontSize: "0.813rem", color: "#4A5567" }} className="nfd-leading-5 nfd-w-5/6">
+                            {__("Give your online store a boost with our all-in-one eCommerce bundle filled with exclusive plugins and tools designed to help you sell your products and services like a pro.", "wp-module-ecommerce")}
+                        </p>
+                    </div>
+                    <div className="nfd-flex-none nfd-mt-8 nfd-self-end">
+
+
+                        <Button
+                            as="a"
+                            className="nfd-button nfd-button--upsell nfd-w-36"
+                            variant="upsell"
+                            data-ctb-id={"f95ccf1e-3028-4ea7-b2c2-847969348e8b"}   
+                            target="_blank"
+                            rel="noopener"
+                            id="buy-ecommerce-addon-btn"
+                        >
+                            <LockIcon className="nfd-w-5 nfd-h-5 nfd--ml-1 nfd-shrink-0" />
+                            {__("Buy Now", "wp-module-ecommerce")}
+                        </Button>
+
+
+                    </div>
+                </div>
+
+                <h2 className="nfd-text-lg nfd-font-medium nfd-mt-8" style={{ color: "#000000" }}>
+                    {__("Exclusive YITH plugins and tools to help you sell online", "wp-module-ecommerce")}
+                </h2>
+                <p style={{ fontSize: "0.813rem", color: "#4A5567" }} className="nfd-leading-5 nfd-w-4/5">
+                    {__("Our eCommerce bundle includes a comprehensive suite of advanced tools designed to boost the performance of your WooCommerce store. This bundle includes these exclusive YITH plugins:", "wp-module-ecommerce")}
+                </p>
+                <div className="nfd-flex nfd-flex-wrap nfd-gap-6">
+                    <div className="nfd-flex-1">
+                        <div className="nfd-flex nfd-flex-wrap nfd-gap-x-10 nfd-mt-3 nfd-pt-0.5 nfd-w-3/4">
+                            <ul className="nfd-flex-1" style={{ color: "#000" }}>
+                                <li className="nfd-flex">
+                                    <CheckMarkIcon className="nfd-mt-0.5 nfd-pt-px nfd-mr-1" /><span>{__("Gift Cards", "wp-module-ecommerce")}</span>
+                                </li>
+                                <li className="nfd-flex">
+                                    <CheckMarkIcon className="nfd-mt-0.5 nfd-pt-px nfd-mr-1" /><span>{__("Wishlists", "wp-module-ecommerce")}</span>
+                                </li>
+                                <li className="nfd-flex">
+                                    <CheckMarkIcon className="nfd-mt-0.5 nfd-pt-px nfd-mr-1" /><span>{__("Custom 'My Account' Page", "wp-module-ecommerce")}</span>
+                                </li>
+                            </ul>
+                            <ul className="nfd-flex-1" style={{ color: "#000" }}>
```

Comment:
can we use class instead of inline style

Reply:
done